### ISSUE TITLE
Running an unknown version

### ISSUE THREAD

**Issue by my95z34** [Tautulli/Tautulli#368]
So, I've got Plex and PlexPy running in the same jail on my FreeNas server.  After fighting with init stuff trying to get PlexPy to run on boot, I gave up and just added it to my crontab 

> @reboot python2 /usr/local/share/plexpy/PlexPy.py -d

which is working great.  Except I get a pop up that says I'm running an unknown version, but the settings screen shows the version properly.  If I close out of the pop up, it comes back after a little while.

See:

> http://imgur.com/dH2htia

Is this because of how I'm running it?  Or something else?  Thanks in advance!

**Comment by JonnyWong16** [comment]
Check your folder permissions.

My guide for running PlexPy as a service on FreeNAS:
http://forums.plex.tv/discussion/comment/1056738/#Comment_1056738

**Comment by my95z34** [comment]
Alrighty, I got it set up using your guide but running service plexpy start doesn't start it as a daemon, instead it just outputs right there in the terminal.  Though, if I restart the jail, it starts just fine.

Also, I still am getting the message that I'm running an unknown version.  AFAIK, it's being ran by the root user, so I chown'd it to root.  Is there something else I should try? 

Thanks!

**Comment by JonnyWong16** [comment]
Try running each of these commands from the PlexPy folder

UPDATED:

```
git remote set-url origin https://github.com/drzoidberg33/plexpy.git
git fetch origin
git reset --hard origin/master
```

OLD:

```
git remote add upstream https://github.com/drzoidberg33/plexpy.git
git fetch upstream
git checkout master
git branch -u upstream/master
git reset --hard upstream/master
git pull
```

**Comment by my95z34** [comment]
Ok, did all that and we appear to be in good shape as far as the notification, so far.

Still get the output in the terminal when running 

> service plexpy start

**Comment by JonnyWong16** [comment]
Only thing I can think of is removing the script and following the steps again.
To remove the script `rm /usr/local/etc/rc.d/plexpy`.

**Comment by my95z34** [comment]
Damn, it just came back.  So, I may just try to put it into its own jail.  Is there an easy way to backup my current data (graphs, etc) so I can restore it in another installation?

Thanks!

**Comment by JonnyWong16** [comment]
All you need to backup is the `plexpy.db` file. Just put the file back before you start up the new PlexPy installation.

**Comment by my95z34** [comment]
Perfect.  Thanks a ton!  Support is absolutely amazing, I really appreciate
it.

**Comment by my95z34** [comment]
Hey, @JonnyWong16, In your link above about how to set it up as a service in FreeNAS, you mention changing line 32 to his username.  Should I be changing that to root, since root will be running everything?

**Comment by JonnyWong16** [comment]
Yes.

**Comment by my95z34** [comment]
I got it working in its own jail.  Thought I should mention a couple issues I had, not necessarily with PlexPy, but with getting it running properly.  I had to symlink python2 to /usr/bin, I think it defaults to /usr/local/bin or something like that.  Also, your default script that goes into /usr/local/etc/rc.d/, I had to change something, but I can't remember now.  I could have sworn it was line #41, python > python2, but I just looked and it's correct.  Hmm.  If I remember I'll comment back.  Until then, thank you for your help!
